File: software/examples/csharp/ExampleSimpleVoltage.cs
using Tinkerforge;

class Example
{
	private static string HOST = "localhost";
	private static int PORT = 4223;
	private static string UID = "XYZ"; // Change to your UID

	static void Main()
	{
		IPConnection ipcon = new IPConnection(); // Create IP connection
		BrickletIndustrialAnalogOut iao = new BrickletIndustrialAnalogOut(UID, ipcon); // Create device object

		ipcon.Connect(HOST, PORT); // Connect to brickd
		// Don't use device before ipcon is connected

		// Set output voltage to 3.3V
		iao.SetVoltage(3300);
		iao.Enable();

		System.Console.WriteLine("Press enter to exit");
		System.Console.ReadLine();
		iao.Disable();
		ipcon.Disconnect();
	}
}

File: software/examples/rust/example_simple_voltage.rs
use std::{error::Error, io};

use tinkerforge::{industrial_analog_out_bricklet::*, ip_connection::IpConnection};

const HOST: &str = "localhost";
const PORT: u16 = 4223;
const UID: &str = "XYZ"; // Change XYZ to the UID of your Industrial Analog Out Bricklet.

fn main() -> Result<(), Box<dyn Error>> {
    let ipcon = IpConnection::new(); // Create IP connection.
    let iao = IndustrialAnalogOutBricklet::new(UID, &ipcon); // Create device object.

    ipcon.connect((HOST, PORT)).recv()??; // Connect to brickd.
                                          // Don't use device before ipcon is connected.

    // Set output voltage to 3.3V
    iao.set_voltage(3300).recv()?;
    iao.enable().recv()?;

    println!("Press enter to exit.");
    let mut _input = String::new();
    io::stdin().read_line(&mut _input)?;

    iao.disable().recv()?;

    ipcon.disconnect();
    Ok(())
}
